src/routes/AgreementEditRoute.js

```diff
@@ -242,6 +278,28 @@ class AgreementCreateRoute extends React.Component {
       .some(r => r.isPending);
   }
 
+  getRecord = (id, resourceType) => {
+    return get(this.props.resources, `${resourceType}.records`, [])
+      .find(i => i.id === id);
+  }
+
+  getOrgs = () => {
+    const { resources } = this.props;
+    const agreement = get(resources, 'agreement.records[0]', {
+      orgs: [],
+    });
+
+    const orgs = agreement.orgs.map(o => ({
+      ...o,
+      interfaces: get(o, 'org.orgsUuid_object.interfaces', [])
+        .map(id => this.getRecord(id, 'interfaces') || id)
+    }));
+
+    return {
```

Comment:
Why would you return an object here? [It makes your life more difficult here](https://github.com/folio-org/ui-agreements/pull/223/files#diff-ea25d95c42ea68fb645dd6525a6d63a4R23) and breaks with the pattern used in all the other `data` properties.